app.py

```diff
@@ -156,7 +157,7 @@ def main():
             schedule=json.dumps(get_schedule(session["username"])),
             days=json.dumps(DAYS),
             components="static/components.html",
-            lunches="[]",
+            lunches=get_lunch_for_date(datetime.date.today()),
```

Comment:
the name for this function confused me, as I figured we'd want to get future lunches too.
maybe we should call it `get_lunches_since_date` and pass in a date that is 28 days in the past.

Reply:
essentially it gets all lunches from 28 days in the past until the last known lunch

Reply:
I fixed this

Reply:
Thanks! I tweaked the name of the argument slightly and committed it to your branch. Go ahead and merge this PR when you get a chance.